.github/workflows/ci.yaml

```diff
@@ -207,18 +207,6 @@ jobs:
           role-to-assume: ${{ secrets.ROLE }}
           role-session-name: credhelper-test
           aws-region: ${{ secrets.REGION }}
-      - name: Remove Finch VM
-        run: |
-          wsl --list --verbose
-          wsl --shutdown
-          wsl --unregister lima-finch
-          wsl --list --verbose
-      - name: Clean up previous files
-        run: |
-          Remove-Item C:\Users\Administrator\.finch -Recurse -ErrorAction Ignore
-          Remove-Item C:\Users\Administrator\AppData\Local\.finch -Recurse -ErrorAction Ignore
-          make clean
```

Comment:
I think make clean still need to execute before build project, I mean this needs to be done both places before and after

Reply:
Everything should already have been made clean from the `Remove Finch VM and Clean Up Previous Environment` in the previous run

Reply:
But it's not too much to add and isn't costly, so I'll take your suggestion just to be safe :)